src/main/java/frc/robot/Actions/ForwardShooter.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.Actions;
+
+import com.flash3388.flashlib.scheduling.ActionControl;
+import com.flash3388.flashlib.scheduling.FinishReason;
+import com.flash3388.flashlib.scheduling.actions.ActionBase;
+import frc.robot.Subsystems.ShooterSystem;
+
+public class ForwardShooter extends ActionBase {
+    private ShooterSystem shooter;
+
+    public ForwardShooter(ShooterSystem shooter){
+        this.shooter = shooter;
+        requires(shooter);
+    }
+
+    @Override
+    public void initialize(ActionControl control) {
+
+    }
+
+    @Override
+    public void execute(ActionControl actionControl) {
+        shooter.move(0.2);
```

Comment:
Use constant speed and pid in the shooterSystem

Reply:
@Maayan-Luzon Can you provide more info on the request for PID?

- What is the wanted setpoint?
- Should the action end when reaching the setpoint?